### PR TITLE
Build updates

### PR DESCRIPTION
**Pull request by jsolahr**
Made additional changes to the following:  
**main.handlebars** = added css reference since we will be using multiple css files.  
**html-routes.js** = added build.css reference to build.handlebars file 
**build.handlebars** = added elements and formatting for build your own theme page
**build.html** = removed file since we' don't need it. it was just a placeholder

### CODE REVIEW

File: views/build.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <title>Build Your Own Theme</title>
+
+    <!-- Bootstrap CSS CDN -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
```
Comment (btgoligowski24) [inline views/build.html]: This shouldn't be needed if our `main.handlebars` has it listed there.

File: views/build.html [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+    <!--Font Awesome-->
+    <script src="https://use.fontawesome.com/5ef15ace37.js"></script>
+
+    <!-- Custom CSS -->
+    <link href="../public/styles/build.css" type="text/css" rel="stylesheet" media="screen,projection" />
```
Comment (btgoligowski24) [inline views/build.html]: this should likely be `href="styles/build.css"`

File: views/build.html (hunk 1 above)
Comment (jsolahr) [reply views/build.html]: This file was removed.

File: views/build.html (hunk 2 above)
Comment (jsolahr) [reply views/build.html]: This file was removed.

File: views/build.html [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+                            crossorigin="anonymous"></script>
+                        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+                            integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+                            crossorigin="anonymous"></script>
+                        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+                            integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+                            crossorigin="anonymous"></script>
+
+
+                        <script src="../public/js/build.js"></script>
```
Comment (btgoligowski24) [inline views/build.html]: any particular reason these are down here instead of the head?

Comment (btgoligowski24) [inline views/build.html]: this likely becomes `src="js/build.js"` and if not, you could likely wrap the file in a document ready function if you're worries about it loading before the page has rendered.

File: views/build.handlebars [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+                    <!--Scripts-->
+                     <script src="/js/build.js"></script>
+                    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+                        integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+                        crossorigin="anonymous"></script>
+                    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+                        integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+                        crossorigin="anonymous"></script>
+                    <script type="text/javascript"
+                        src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (btgoligowski24) [inline views/build.handlebars]: see comments on other commit referencing this section

File: views/build.html (hunk 1 above)
Comment (btgoligowski24) [reply views/build.html]: yeah, i just realized this was the template file... der.. sorry.

File: views/build.html (hunk 2 above)
Comment (btgoligowski24) [reply views/build.html]: git, sorry, was just starting at furthest backing and working forward. trying to review via FIFO.

File: views/layouts/main.handlebars [hunk 5]
```diff
@@ -3,9 +3,23 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css"> 
     <link rel="stylesheet" href="/styles/styles.css" media="screen" title="no title" charset="utf-8">
-   
+
+      <!-- Bootstrap CSS CDN -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
```
Comment (btgoligowski24) [inline views/layouts/main.handlebars]: these two are duplicated

File: views/layouts/main.handlebars [hunk 6]
```diff
@@ -3,21 +3,24 @@
...
-
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
         integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-
     <!--Font Awesome-->
     <script src="https://use.fontawesome.com/5ef15ace37.js"></script>
-
     <!-- Custom CSS -->
     <link rel="stylesheet" href="styles/{{style}}" media="screen" title="no title" charset="utf-8">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css">
```
Comment (btgoligowski24) [inline views/layouts/main.handlebars]: this is duplicated with line 7 and an older version than line 11

File: views/layouts/main.handlebars [hunk 7]
```diff
@@ -3,21 +3,24 @@
 
 <head>
     <meta charset="utf-8">
+    
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css">
```
Comment (btgoligowski24) [inline views/layouts/main.handlebars]: this is duplicated with line 17 and an older version than line 11

File: views/layouts/main.handlebars [hunk 8]
```diff
@@ -3,21 +3,24 @@
...
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css"> 
-    <link rel="stylesheet" href="/styles/styles.css" media="screen" title="no title" charset="utf-8">
-
-      <!-- Bootstrap CSS CDN -->
+    <!-- Bootstrap CSS CDN -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
         integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
         integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
```
Comment (btgoligowski24) [inline views/layouts/main.handlebars]: we're switching to a non-minified version of the code here in another commit.

<script src="https://code.jquery.com/jquery-3.4.1.js" integrity="sha256-WpOohJOqMqqyKL9FccASB9O0KwACQJpFTUBLTYOVvVU=" crossorigin="anonymous"></script>

File: views/build.html (hunk 3 above)
Comment (jsolahr) [reply views/build.html]: This is the build.html file. It will be deleted.

Comment (jsolahr) [reply views/build.html]: This is the build.html file that I commited to be deleted

Comment (btgoligowski24) [reply views/build.html]: true, but this is still relevant for the build.handlebars file.

File: views/build.handlebars (hunk 4 above)
Comment (btgoligowski24) [reply views/build.handlebars]: these also shouldn't be needed since we've got them in the main.handlebars file. so it will be inherited to here as well.

Comment (jsolahr) [reply views/build.handlebars]: good call, I'll remove.

**Pull request merged**